packages/devextreme/testing/testcafe/tests/scheduler/appointmentForm/showAppointmentPopup.ts

```diff
@@ -29,6 +29,30 @@ test('Invoke showAppointmentPopup method shouldn\'t raise error if value of curr
   height: 600,
 }));
 
+test('Appointment popup shouldn\'t raise error if appoitment is recursive', async (t) => {
```

Comment:
Maybe the name of test shouldn't focus on the error? Test that the form works in case that the time zone exisit and recurrence

Reply:
Test is specially to check that there are no errors, and we need to check that there's no error in the console, so I think it's okay to have this name